src/SDKs/SqlManagement/Management.Sql/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -56,6 +56,8 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("Sql", "Jobs", "2017-03-01-preview"),
                 new Tuple<string, string, string>("Sql", "LongTermRetentionBackups", "2017-03-01-preview"),
                 new Tuple<string, string, string>("Sql", "ManagedDatabases", "2017-03-01-preview"),
+                new Tuple<string, string, string>("Sql", "ManagedInstanceEncryptionProtectors", "2017-10-01-preview"),
+                new Tuple<string, string, string>("Sql", "ManagedInstanceKeys", "2017-10-01-preview"),
```

Comment:
Please run `msbuild build.proj /t:Build /p:Scope=SDKs\SqlManagement` and check in any changes generated

Reply:
If there's nothing generated, we are good.